README.md

```diff
@@ -9,76 +9,92 @@ SAML: https://wiki.oasis-open.org/security/FrontPage
 ECP: http://docs.oasis-open.org/security/saml/Post2.0/saml-ecp/v2.0/saml-ecp-v2.0.html
 
 ## Setup
-To build the project, execute `mvn clean install -nsu` at the project root.
-The `distribution` module will contain a full package of the deployment after the build.
-Tests can be run with the script:
-
-`<PATH_TO_PROJECT>/distribution/command-line/target/command-line-<VERSION>-bin/bin/samltest.sh`.
-
->NOTE
->
->In order for the test kit to execute properly, you must configure both
-the test kit's and your IdP's/SP's metadata, as well as implement plugins
+To build the project, execute `gradle build` at the project root.
+The `distribution/command-line` module will contain a full package of the deployment after the build.
+
+Tests can be run with the `samlconf` script under `<PATH_TO_PROJECT>/distribution/command-line/build/distributions/samlconf-<VERSION>/bin/`.
+Build the project will create the distribution, to unzip it, execute `gradle build installDist`, which will unzip it and place it under `build/install`.
+
+The `samlconf` script takes multiple parameters:
+
+```
+NAME
+       samlconf - Runs the SAML Conformance Tests against an IdP or an SP
+
+SYNOPSIS
+       samlconf [-i path] [--idpMetadata path]
+           [-p path] [--plugins path] 
+
+DESCRIPTION
+       Runs the SAML Conformance Tests which tests the complance of an IdP and/or an SP
+       with the SAML Specifications. If a compliance issue is identified, a 
+       SAMLConformaceException will be throw with an explanation of the error an a direct
```

Comment:
❗️ s/b "SAMLConformanceException" (missing 'n').
❗️ s/b "...will be thrown..." (missing 'n').
❗️ s/b "...error and a direct..." (missing 'd')